docs/content/en/docs/getting-started/observability/_index.md

```diff
@@ -76,12 +76,16 @@ spec:
   keptnAppCreationRequestTimeoutSeconds: 30
 ```
 
-Apply the file and restart Keptn to pick up the new config:
+Apply the file and wait for Keptn to pick up the new configuration:
 
 ```shell
-kubectl apply -f keptnconfig.yaml
+kubectl apply -f collectorconfig.yaml
```

Comment:
Here https://github.com/keptn/lifecycle-toolkit/pull/2326/files#diff-8e804892367287cd549232d551a50ddd7b6f800736a3a27f4fb326d49d7425caL64 you refer to it as `keptnconfig.yaml`, let's do the same here as well 
```suggestion
kubectl apply -f keptnconfig.yaml
```